src/WpBookingsUiModule.php

```diff
@@ -500,25 +501,38 @@ function () use ($c) {
             }
         );
 
-        $aboutContext = [
-            'get_started_url'     => $c->get('wp_bookings_ui/urls/get_started'),
-            'feature_request_url' => $c->get('wp_bookings_ui/urls/feature_request'),
-            'contact_us_url'      => $c->get('wp_bookings_ui/urls/contact_us'),
-            'enter_license_url'   => admin_url($c->get('wp_bookings_ui/urls/license')),
-        ];
         $this->aboutPageId = add_submenu_page(
             $rootMenuConfig->get('menu_slug'),
             $this->__($aboutMenuConfig->get('page_title')),
             $this->__($aboutMenuConfig->get('menu_title')),
             $aboutMenuConfig->get('capability'),
             $aboutMenuConfig->get('menu_slug'),
-            function () use ($c, $aboutContext) {
+            function () use ($c) {
                 $aboutTemplate = $c->get('eddbk_ui_about_template');
-                echo $aboutTemplate->render($aboutContext);
+                echo $aboutTemplate->render($this->_getAboutPageContext($c->get('wp_bookings_ui/urls')));
             }
         );
     }
 
+    /**
+     * Get context for about page.
+     *
+     * @since [*next-version*]
+     *
+     * @param array|stdClass|ContainerInterface $urls Map of urls for context.
+     *
+     * @return array Context for about page.
+     */
+    protected function _getAboutPageContext($urls)
```

Comment:
I was hoping you would just make a separate method that renders the page. The context can be created in that method. What's important is that the concerns are separated. Right now, there are still more reasons for `_adminMenu()` to change than there should be. This method is about the menu, and not about rendering a page.